Error launching: Failed to acquire auth token

I'm getting the following error when I start azbrowse (error message manually clean up as it was overlayed on the UI!)

```
panic: Failed to acquire auth token: invalid character 'A' looking for beginning of value                                                  goroutine 37 [running]:
main.main.func12(0xc0422380f0, 0xc042220120, 0xc0422201b0, 0xc042150200)                                                             
        /go/src/github.com/lawrencegripper/azbrowse/main.go:202 +0x315 
created by main.main                                                   
        /go/src/github.com/lawrencegripper/azbrowse/main.go:194 +0xdc5 
```

--- a/armclient/authutils.go
+++ b/armclient/authutils.go
@@ -13,7 +13,7 @@ type azCLIToken struct {
 }
 
 func aquireTokenFromAzCLI() (azCLIToken, error) {
-	out, err := exec.Command("az", "account", "get-access-token").Output()
+	out, err := exec.Command("az", "account", "get-access-token", "--output", "json").Output()
 	if err != nil {
 		return azCLIToken{}, err
 	}

